app/src/main/java/com/iic/shopingo/ui/request_flow/activities/CreateShoppingListActivity.java

```diff
@@ -49,8 +58,10 @@ protected void onCreate(Bundle savedInstanceState) {
 
   @OnClick(R.id.create_request_create_button)
   public void onCreateRequest(View view) {
-    request.items = itemListView.getAllItems();
-    request.price = Integer.parseInt(priceView.getText().toString());
+    shoppingList.setItems(itemListView.getAllItems());
+    shoppingList.setOffer(Integer.parseInt(priceView.getText().toString()));
+    request.setShoppingList(shoppingList);
+    request.setStatus(BaseRequest.RequestStatus.PENDING);
```

Comment:
But it contains the wrong information. The status is not yet decided (that's up to the server) and the contact is technically illegal (null contact should not really be allowed in a request) when you go to select a new one after cancellation.